client/src/globalState.tsx

```diff
@@ -144,7 +152,7 @@ export const GlobalStateProvider: React.FC<GlobalStateProviderProps> = ({
   hasEverUsedTour,
   visitTourStep,
   children,
-}) => {
+}: GlobalStateProviderProps) => {
```

Comment:
I think if you do something like the following you can avoid the need for an explicit `children: ReactNode;` in the `GlobalStateProviderProps` interface...
```suggestion
}: PropsWithChildren<GlobalStateProviderProps>) => {
```

Alternatively, rather than `interface GlobalStateProviderProps {` further up, I think you could write `type GlobalStateProviderProps = PropsWithChildren<{`

Reply:
Ah yeah, that seems to work! Good shout, thanks